src/pages/groupComparison/AlterationEnrichmentTypeSelector.tsx

```diff
@@ -0,0 +1,703 @@
+        this.currentSelectedMutations.nonsense_mutation = value;
+        this.currentSelectedMutations.nonsense = value;
+        this.currentSelectedMutations.stopgain_snv = value;
+        this.currentSelectedMutations.stop_gained = value;
+        this.currentSelectedMutations.splice_site = value;
+        this.currentSelectedMutations.splice = value;
+        this.currentSelectedMutations.splicing = value;
+        this.currentSelectedMutations.splice_site_snp = value;
+        this.currentSelectedMutations.splice_site_del = value;
+        this.currentSelectedMutations.splice_site_indel = value;
+        this.currentSelectedMutations.splice_region = value;
+        this.currentSelectedMutations.splice_region_variant = value;
+        this.currentSelectedMutations.translation_start_site = value;
+        this.currentSelectedMutations.initiator_codon_variant = value;
+        this.currentSelectedMutations.start_codon_snp = value;
+        this.currentSelectedMutations.start_codon_del = value;
+        this.currentSelectedMutations.nonstop_mutation = value;
+        this.currentSelectedMutations.stop_lost = value;
+        this.currentSelectedMutations.inframe_del = value;
+        this.currentSelectedMutations.inframe_deletion = value;
+        this.currentSelectedMutations.in_frame_del = value;
+        this.currentSelectedMutations.in_frame_deletion = value;
+        this.currentSelectedMutations.indel = value;
+        this.currentSelectedMutations.inframe_ins = value;
+        this.currentSelectedMutations.inframe_insertion = value;
+        this.currentSelectedMutations.in_frame_ins = value;
+        this.currentSelectedMutations.in_frame_insertion = value;
+        this.currentSelectedMutations.nonframeshift_deletion = value;
+        this.currentSelectedMutations.nonframeshift = value;
+        this.currentSelectedMutations.nonframeshift_insertion = value;
+        this.currentSelectedMutations.inframe = value;
+        this.currentSelectedMutations.truncating = value;
+        this.currentSelectedMutations.feature_truncation = value;
+        this.currentSelectedMutations.fusion = value;
+        this.currentSelectedMutations.silent = value;
+        this.currentSelectedMutations.synonymous_variant = value;
+        this.currentSelectedMutations.targeted_region = value;
+        this.currentSelectedMutations.any = value;
+        this.currentSelectedMutations.other = value;
+    }
+
+    @autobind
+    private onInputClick(event: React.MouseEvent<HTMLInputElement>) {
+        switch ((event.target as HTMLInputElement).value) {
+            case 'HOMDEL':
+                this.currentSelectedCopyNumber.HOMDEL = !this
+                    .currentSelectedCopyNumber.HOMDEL;
+                break;
+            case 'AMP':
+                this.currentSelectedCopyNumber.AMP = !this
+                    .currentSelectedCopyNumber.AMP;
+                break;
+            case 'missense':
```

Comment:
a switch statement where the cases are string values instead of enum doesn't smell great to me..

Reply:
Updated